content/kubernetes/release-notes/k8s-6-2-8-10-2022-01.md

```diff
@@ -0,0 +1,128 @@
+---
+Title: Redis Enterprise for Kubernetes Release Notes 6.2.8-10 (January 2022)
+linktitle: 6.2.8-10 (January 2022)
+description: Support for Istio as ingress controller, K8s 1.22 (AKS, kOps, GKE), OpenShift 4.9. 
+weight: 66
+alwaysopen: false
+categories: ["Platforms"]
+aliases: [
+    /kubernetes/release-notes/k8s-6-2-8-10-2022-01.md,
+    /kubernetes/release-notes/k8s-6-2-8-10-2022-01/,   ]
+---
+## Overview
+
+The Redis Enterprise [K8s 6.2.8-10](https://github.com/RedisLabs/redis-enterprise-k8s-docs/releases/tag/v6.2.8-10) release provides support for the [Redis Enterprise Software release 6.2.8]({{<relref "/rs/release-notes/rs-6-2-8-october-2021.md">}}) and includes several enhancements and bug fixes.
```

Comment:
The link [K8s 6.2.8-10](https://github.com/RedisLabs/redis-enterprise-k8s-docs/releases/tag/v6.2.8-10) gives me a 404.

Reply:
This will be resolved when @amiramm creates the release tag on their github project.